ENGAGEHF/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+//  DashboardView.swift
+//  ENGAGEHF
+//
+//  Created by Nick Riedman on 4/3/24.
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct Greeting: View {
+    @State private var dateString: String?
+    
+    var body: some View {
+        HStack {
+            Text("Hello, world!")
+                .font(.title2)
+                .accessibilityLabel(Text("DASHBOARD_GREETING"))
+            Spacer()
+            Text(dateString ?? "No date")
```

Comment:
I am not quite sure what you aim to achieve here, but if you just want to show the current date you can use something like
```swift
Text(.now, style: .date)
```

Without using any state or task to kick things off.

I can recommend looking at https://developer.apple.com/documentation/swiftui/text and https://www.hackingwithswift.com/quick-start/swiftui/how-to-format-dates-inside-text-views for more context.